src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoresAndReports.tsx

```diff
@@ -38,6 +38,7 @@ export const ScoresAndReports = () => {
 
   const sections: ActivitySettingsSection[] = watch(sectionsName);
   const scores: ActivitySettingsScore[] = watch(scoresName);
+  const isCheckboxesEnabled = scores?.length || sections?.length;
```

Comment:
```suggestion
  const isCheckboxesEnabled = scores?.length || sections?.length;
```
const isCheckboxesDisabled = !(scores?.length || sections?.length);